### PR TITLE
Added Host Next TOC form

### PR DESCRIPTION
**Pull request by Dvessen83**
Added Host Next TOC form
#35

### CODE REVIEW

File: app/assets/stylesheets/pages.scss [hunk 1]
```diff
@@ -32,3 +32,8 @@ footer{
   text-align: center;
   margin-top: 20px;
 }
+
+.commentbox{
+  height: 200px !important;
```
Comment (arnoFleming) [inline app/assets/stylesheets/pages.scss]: why so `!important`? remove em! this makes overriding styles a nightmare

File: app/assets/stylesheets/pages.scss [hunk 2]
```diff
@@ -32,3 +32,8 @@ footer{
...
+  vertical-align: top !important;
```
Comment (arnoFleming) [inline app/assets/stylesheets/pages.scss]: idem

File: app/views/shared/_host_toc_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<section>
+  <div align="center">
+    <h3>Host next TOC</h3>
+    <h4>Talk to us</h4>
+  </div>
+    <form>
```
Comment (arnoFleming) [inline app/views/shared/_host_toc_form.html.erb]: You'll link these to an `InCompanyRequest` later, I hope?

Comment (arnoFleming) [inline app/views/shared/_host_toc_form.html.erb]: If yes, then don't add it here :)

Comment (Dvessen83) [inline app/views/shared/_host_toc_form.html.erb]: Yes! But model doesn't or didn't exist yet at this time.

**Comment by arnoFleming** [comment]
👍

**Pull request merged**